1-WebApp-OIDC/1-5-B2C/Views/Shared/_Layout.cshtml

```diff
@@ -38,13 +38,11 @@
         </div>
     </nav>
 
-    <partial name="_CookieConsentPartial" />
```

Comment:
>_CookieConsentPartial [](http://example.com/codeflow?start=19&length=21)

I think the method call
app.UseCookiePolicy(); 
in startup.cs is also related and please go ahead and remove it from the code as well. thnx